miriway

```diff
@@ -1,6 +1,25 @@
 #! /bin/sh
 set -e
 
+### Gtk Theming Tweaks
+if command -v gsettings > /dev/null then
+        gtk30_config="${XDG_CONFIG_HOME:-$HOME/.config}/gtk-3.0/settings.ini"
+        if [ -f "$gtk30_config" ]; then
+          cursor_theme="$(grep 'gtk-cursor-theme-name' "$gtk30_config" | sed 's/.*\s*=\s*//')"
+          font_name="$(grep 'gtk-font-name' "$gtk30_config" | sed 's/.*\s*=\s*//')"
+          gtk_theme="$(grep 'gtk-theme-name' "$gtk30_config" | sed 's/.*\s*=\s*//')"
+          icon_theme="$(grep 'gtk-icon-theme-name' "$gtk30_config" | sed 's/.*\s*=\s*//')"
+  
+          gsettings set org.gnome.desktop.interface gtk-theme "$gtk_theme"
+          gsettings set org.gnome.desktop.interface icon-theme "$icon_theme"
+          gsettings set org.gnome.desktop.interface cursor-theme "$cursor_theme"
+          gsettings set org.gnome.desktop.interface font-name "$font_name"
+        fi
+fi
+
+### Window Buttons Fix
+gsettings set org.gnome.desktop.wm.preferences button-layout ":minimize,maximize,close"
```

Comment:
This should be condition for gsettings being installed.

Reply:
Ok !!!